src/index.ts

```diff
@@ -41,6 +40,7 @@ if (config.health.healthz.enabled) {
     Healthz.listen();
 }
 
+let bot: Mjolnir | null = null;
 (async function () {
```

Comment:
Rather than making `bot` a global variable, I'd suggest

replacing the the call to method `catch` with a

```ts
try {
  const storagePath = ..
  // ...
  await bot.start();
} catch (err) {
  bot?.logMesage(...);
  process.exit(1);
}
```